SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -281,10 +303,13 @@ public static PersonOtherName CreatePersonOtherName(long? personId = null)
             int? workerId = null
         )
         {
+            var team = CreateTeam();
+
             return new Faker<WorkerTeam>()
                 .RuleFor(t => t.Id, f => f.UniqueIndex)
                 .RuleFor(t => t.WorkerId, f => workerId ?? f.UniqueIndex)
-                .RuleFor(t => t.TeamId, f => f.UniqueIndex);
+                .RuleFor(t => t.TeamId, f => f.UniqueIndex)
```

Comment:
QQ: When should we just use `f.UniqueIndex` or `f.UniqueIndex + 1`?

Reply:
UniqueIndex seems to start at 0 (and increments by 1 every time it is assigned), there are cases in our database where the minimum accepted value is 1 so the `f.UniqueIndex + 1` is to help some GatewayTests in which we actually add something to PostgreSQL.

It would probably be nice to have this consistently everywhere. Clearly it's also a little confusing so could be documented why.